Assets/MixedRealityToolkit/_Core/Definitions/MixedRealityConfigurationProfile.cs

```diff
@@ -220,29 +220,5 @@ void ISerializationCallbackReceiver.OnAfterDeserialize()
         }
 
         #endregion  ISerializationCallbackReceiver Implementation
-
-        #region Mixed Reality Controller Mapping helpers
-
-        // TODO - needs validation to ensure duplicates are not added?
-
-        public MixedRealityControllerMapping GetControllerMapping(Type controllerType, Handedness hand)
```

Comment:
How is this fixed by removing it>?

Reply:
It's moved into the implemented controllers class